static/js/publisher/release/components/progressiveBar.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class ProgressiveBar extends React.Component {
```

Comment:
Looking at the evolution of release UI components and recent refactorings it would be good to split this into smaller "view-only" component and "functionality" component that renders it.

In this case I suppose there would be a ProgressiveBar component that only recieves props telling it what to render (percentage). The other component will handle mouse changes and propagate them to the view one.
This would make progressive bar view more reusable in different context and hopefully make all components more focused.